src/eei.cpp

```diff
@@ -503,10 +495,6 @@ namespace HeraVM {
       context->fn_table->call(&call_result, context, &call_message);
 
       if (call_result.output_data) {
-        vector<uint8_t> result(call_result.output_data, call_result.output_data + call_result.output_size);
-        result.resize(resultLength);
-        storeMemory(result, 0, resultOffset, resultLength);
-
         lastReturnData.assign(call_result.output_data, call_result.output_data + call_result.output_size);
```

Comment:
You can do it unconditionally now. It will handle case of (null, 0) as well.

Reply:
I think I'll leave it for the moment. Do you want to create a new PR which changes to in all appropriate places?